cmd/api-linter/github_actions.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/googleapis/api-linter/lint"
+)
+
+// printGithubActions returns lint errors in GitHub actions format.
+func printGithubActions(responses []lint.Response) ([]byte, error) {
```

Comment:
Nit: s/printGithubActions/formatGitHubActionOutput/

Reply:
If that name isn't accepted, please at least capitalize "GitHub" accurately.

Reply:
Done